GeofenceDisplay/src/geofence-display.ts

```diff
@@ -2,153 +2,82 @@ export {}
 
 import L from 'leaflet'
 import './geofence-display.css'
+import './rastercoords.js'
 
 const PlantquestGeofenceDisplay = L.Layer.extend({
-  options: {
-    // geofenceLabelPane.style.zIndex = 235
-    // provide geofence data to component using an option
-  },
-
   initialize: function (this: any, options: any) {
-    console.log('PGeoD init')
+    console.log('GeofenceDisplay init function called.')
+    L.Util.setOptions(this, options)
     this._state = {
```

Comment:
use https://github.com/rjrodger/gubu to validate these options

Reply:
👍